index.js

```diff
@@ -77,14 +77,11 @@ class Policies {
       (ctx.state.user[this.config.twoFactorEnabled] && !ctx.session.otp)
     ) {
       ctx.session.returnTo = ctx.originalUrl || ctx.req.url;
-      if (!ctx.is('json'))
-        ctx.flash(
-          'warning',
-          ctx.translate
-            ? ctx.translate('TWO_FACTOR_REQUIRED')
-            : 'Please log in with two factor authentication' +
-                ' to view the page you requested.'
-        );
+      const message = ctx.translate
+        ? ctx.translate('TWO_FACTOR_REQUIRED')
+        : 'Please log in with two factor authentication ' +
```

Comment:
Don't use `+` to do String joining over multiple lines, just make it one line, and the auto-formatter with prettier/xo will do the rest of the work for you.  This is super bad practice.

Reply:
It looks like prettier rules of `max-len` were [previously setup to error](https://github.com/ladjs/policies/blob/7640b259acc82ba46912b50c1cac99134385c031/package.json#L98-L104).

```
> @ladjs/policies@1.0.8 lint /Users/swarman/Development/Source/side/policies
> xo && remark . -qfo


  index.js:82:1
  ✖  82:1  This line has a length of 89. Maximum allowed is 80.  max-len
```